### PR TITLE
DDCE-4194: Update CSP config

### PR DESCRIPTION
**Pull request by MRBeresford**

### CODE REVIEW

**Review by matthewwardcap — CHANGES_REQUESTED** [review]
Good few minor comments:

1) in run_all_tests.sh change `it:test` to `IntegrationTest/test`

2) In build.sbt line 76. Change `addCommandAlias("scalastyleAll", "all scalastyle test:scalastyle it:scalastyle")` to `addCommandAlias("scalastyleAll", "all scalastyle Test/scalastyle IntegrationTest/scalastyle")`
also change coverage from 70 to 93

3) in plugins.sbt remove line 3, replace lines 1/2 with:
`resolvers += MavenRepository("HMRC-open-artefacts-maven2", "https://open.artefacts.tax.service.gov.uk/maven2")`
`resolvers += Resolver.url("HMRC-open-artefacts-ivy2", url("https://open.artefacts.tax.service.gov.uk/ivy2"))(Resolver.ivyStylePatterns)`

File: conf/application.conf [hunk 1]
```diff
@@ -64,14 +42,6 @@ play.modules.enabled += "config.Module"
 play.i18n.langs = ["en", "cy"]
 
 microservice {
-  metrics {
-    graphite {
-      host    = localhost
-      port    = 2003
-      prefix  = play.${appName}.
-      enabled = true
```
[the remark this answers is not in the document]
Comment (MRBeresford) [reply conf/application.conf:72]: I used to leave `enabled=true` in here but is this doing anything locally? its overwritten in all other envs so do we need it here?

[the remark this answers is not in the document]
Comment (MRBeresford) [reply conf/application.conf:69]: Set to true in app-config-common

**Review by matthewwardcap — APPROVED** [review]
All good to me

**Pull request merged**